lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && ((this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) || routeInfo.urlInfo.query.noop === 'true')) {
```

Comment:
I see. Please rename the option `noop` for consistency with query, and remove `justForward` since it's redundant with noop.

if options.noop || query.noop